packages/utilities/src/dom/raiseClick.ts

```diff
@@ -1,4 +1,6 @@
-/** Raises a click event. */
+/** Raises a click event.
+ * @deprecated
```

Comment:
This function is only used from `FocusZone` so I've opted to move the implementation there and deprecate this.

Reply:
Could you write a deprecation comment for this?

```suggestion
 * @deprecated Moved to `FocusZone` component since it was the only place internally using this function.
```